pkgs/development/haskell-modules/configuration-ghc-9.6.x.nix

```diff
@@ -88,23 +86,20 @@ self: super: {
   # allows base >= 4.18
   tasty-hedgehog = doDistribute self.tasty-hedgehog_1_4_0_1;
 
+  # v0.1.6 forbids base >= 4.18
+  singleton-bool = assert !(self ? singleton-bool_0_1_7) doDistribute super.singleton-bool_0_1_7;
```

Comment:
What is the purpose of this assert?

Reply:
Missing a semicolon after the assert.

Reply:
Because I want the override to be removed if Stackage upgrades to singleton-bool-0.1.7 or greater.

Reply:
Yeah, but then the singleton-bool_0_1_7 attribute will disappear anyway and this will throw an eval error even without an assert.

Reply:
Ok, I have removed this.

Reply:
> Yeah, but then the singleton-bool_0_1_7 attribute will disappear anyway and this will throw an eval error even without an assert.

I mean it's kinda self documenting that the override would then need to be removed, but a comment is also ok probably.